src/widgetastic/browser.py

```diff
@@ -751,6 +751,18 @@ def send_keys_to_focused_element(self, *keys):
         """
         ActionChains(self.selenium).send_keys(*keys).perform()
 
+    def copy(self):
```

Comment:
Looking at the current API for browser classes, I think this should be named `copy_focused_element`/`paste_to_focused_element` if they aren't going to take a locator to resolve which element to copy from or paste to.

Alternatively, which would obviously complicate the method, it could take a locator and resolve which element to focus before selecting/copying/pasting.